src/exts/tvdb_info/main.py

```diff
@@ -0,0 +1,126 @@
+        self.results: SearchResults = results
+        self.dropdown = discord.ui.Select(
+            placeholder="Not what you're looking for? Select a different result.",
+            options=[
+                discord.SelectOption(
+                    label=result.name or result.title or "",
+                    value=str(i),
+                    description=result.overview[:100] if result.overview else None,
+                )
+                for i, result in enumerate(self.results)
+            ],
+        )
+        self.dropdown.callback = self._dropdown_callback
+        self.add_item(self.dropdown)
+        self.index = 0
+
+    def _get_embed(self) -> discord.Embed:
+        result = self.results[self.index]
+        original_title = result.name
+        en_title = result.translations["eng"]
+        original_overview = result.overview
+        en_overview = result.overviews["eng"]
+        if en_overview and en_overview != original_overview:
+            overview = f"{en_overview}"
+        elif not en_overview and original_overview:
+            overview = f"{original_overview}\n\n*No English overview available.*"
+        else:
+            overview = "*No overview available.*"
+        if original_title and original_title != en_title:
+            title = f"{original_title} ({en_title})"
+        else:
+            title = en_title
+        if result.id[0] == "m":
+            title = f"{MOVIE_EMOJI} {title}"
+            url = f"https://www.thetvdb.com/movies/{result.slug}"
+        else:
+            title = f"{SERIES_EMOJI} {title}"
+            url = f"https://www.thetvdb.com/series/{result.slug}"
+        embed = discord.Embed(title=title, color=discord.Color.blurple(), url=url)
+        embed.add_field(name="Overview", value=overview, inline=False)
+        embed.set_footer(text=THETVDB_COPYRIGHT_FOOTER, icon_url=THETVDB_LOGO)
+        embed.set_image(url=result.image_url)
+        return embed
+
+    async def _dropdown_callback(self, interaction: discord.Interaction) -> None:
+        if not self.dropdown.values or not isinstance(self.dropdown.values[0], str):  # noqa: PD011
+            log.error("Dropdown values are empty or not a string but callback was triggered.")
+            return
+        self.index = int(self.dropdown.values[0])  # noqa: PD011
+        if not self.message:
+            log.error("Message is not set but callback was triggered.")
+            return
+        await self.message.edit(embed=self._get_embed(), view=self)
+        await interaction.response.defer()
+
+    async def send(self, ctx: ApplicationContext) -> None:
+        """Send the view."""
+        await ctx.respond(embed=self._get_embed(), view=self)
+
+
+class InfoCog(Cog):
+    """Cog to verify the bot is working."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @slash_command(
+        name="search",
+        description="Search for a movie or series.",
```

Comment:
These are redundant, by default, the name of the command will be the function name and the description will be the first line from the docstring. Applies for all occurences.